target/linux/mediatek/dts/mt7986a-arcadyan-wg620443.dts

```diff
@@ -0,0 +1,385 @@
+// SPDX-License-Identifier: (GPL-2.0 OR MIT)
```

Comment:
Chechpatch warnings: 
~~~
$ ~/workspace/linux/scripts/checkpatch.pl --file --strict mt7986a-arcadyan-wg620443.dts 

CHECK: Please don't use multiple blank lines
#51: FILE: mt7986a-arcadyan-wg620443.dts:51:
+
+

ERROR: trailing whitespace
#141: FILE: mt7986a-arcadyan-wg620443.dts:141:
+^I^I$

ERROR: code indent should use tabs where possible
#358: FILE: mt7986a-arcadyan-wg620443.dts:358:
+^I^I ^Ipartition@5380000 {$

WARNING: please, no space before tabs
#358: FILE: mt7986a-arcadyan-wg620443.dts:358:
+^I^I ^Ipartition@5380000 {$

ERROR: trailing whitespace
#372: FILE: mt7986a-arcadyan-wg620443.dts:372:
+^I^I^I$

CHECK: Please don't use multiple blank lines
#377: FILE: mt7986a-arcadyan-wg620443.dts:377:
+
+
~~~